Wireframe/index.html

```diff
@@ -1,13 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+
+    <div class="main">
+      <header>
+        <h1>Sustainable Life & Wellness</h1>
+        <h5>"Inspiring Conscious Choices for a Greener Life"</h5>
+      </header>
+      <div class="chaild1">
```

Comment:
It may have been better to use a clearer class name so that others are able to understand better.